routes/interactions.js

```diff
@@ -29,6 +30,16 @@ router.post('/', async (req, res) => {
       };
 
       await shoutOutService.submitDialog(reqInfo);
+    } else if (data.type === 'dialog_submission' && data.callback_id === 'view-shoutout') {
+      const reqInfo = {
+        channelId: data.channel.id,
+        userId: data.submission.user,
+        content: data.submission.ShoutOut,
+        team: data.team.id,
+        user_id: data.user.id,
+      };
+
+      await shoutOutService.getUserShoutOuts(reqInfo);
     }
   } catch (err) {
     res.status(500).send(err);
```

Comment:
The `err` returned as a response may not be descriptive enough for the client, you could dig into the `err` object and send a more appropriate error message as a response.

Reply:
I have changed this to be much more descriptive by providing a message along with an error stack